packages/application/src/services.ts

```diff
@@ -15,22 +15,47 @@
  */
 
 // A list of supported docker images for submission
-// TODO: replace this static list with a call to a metadata service
-const dockerImages: { [key: string]: string } = {
-  'elyra/tensorflow:1.15.2-py3': 'Tensorflow 1.15.2',
-  'elyra/tensorflow:1.15.2-gpu-py3': 'Tensorflow 1.15.2 w/ GPU',
-  'tensorflow/tensorflow:2.0.0-py3': 'Tensorflow 2.0',
-  'tensorflow/tensorflow:2.0.0-gpu-py3': 'Tensorflow 2.0 w/ GPU',
-  'pytorch/pytorch:1.2-cuda10.0-cudnn7-runtime': 'Pytorch 1.2 w/ CUDA-runtime',
-  'pytorch/pytorch:1.2-cuda10.0-cudnn7-devel': 'Pytorch 1.2 w/ CUDA-devel',
-  'amancevice/pandas:1.0.3': 'Pandas 1.0.3'
-};
+import { SubmissionHandler } from './submission';
+
+// const dockerImages: { [key: string]: string } = {
+//   'elyra/tensorflow:1.15.2-py3': 'Tensorflow 1.15.2',
+//   'elyra/tensorflow:1.15.2-gpu-py3': 'Tensorflow 1.15.2 w/ GPU',
+//   'tensorflow/tensorflow:2.0.0-py3': 'Tensorflow 2.0',
+//   'tensorflow/tensorflow:2.0.0-gpu-py3': 'Tensorflow 2.0 w/ GPU',
+//   'pytorch/pytorch:1.2-cuda10.0-cudnn7-runtime': 'Pytorch 1.2 w/ CUDA-runtime',
+//   'pytorch/pytorch:1.2-cuda10.0-cudnn7-devel': 'Pytorch 1.2 w/ CUDA-devel',
+//   'amancevice/pandas:1.0.3': 'Pandas 1.0.3'
+// };
 
 /**
  * A utilities class for various elyra services.
  */
 export class FrontendServices {
-  static getDockerImages(): { [key: string]: string } {
-    return dockerImages;
+  async getMetadata(name: string): Promise<any> {
```

Comment:
What is your reasoning behind changing this from being a util class. Ie. why is this not static?